core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3071,36 +3042,7 @@ class KafkaApis(val requestChannel: RequestChannel,
   }
 
   def handleExpireTokenRequestZk(request: RequestChannel.Request): Unit = {
```

Comment:
Maybe we can remove this method to remove "zk" from this class? in the line#3040 we can rewrite it to `maybeForwardToController(request, KafkaApis.shouldAlwaysForward)`

Reply:
> Maybe we can remove this method to remove "zk" from this class? 

It makes sense to me. Remove all zk-related code from the codebase. I believe this is currently the general consensus. I will address it.